.github/workflows/ci.yml

```
@@ -34,6 +34,11 @@ jobs:
       - uses: Swatinem/rust-cache@v2
       - run: cargo test --workspace --no-run
       - run: cargo test --workspace --no-fail-fast
+      - if: failure()
+        uses: actions/upload-artifact@v4
+        with:
+          name: test-artifacts-${{ matrix.os }}
+          path: tests/*/render/**
```

```suggestion
          path: tests/store/render/**
```
that should be the only folder with a "render" subfolder